### PR TITLE
Update meta tag to include unsplash

### PR DESCRIPTION
**Pull request by taysea**
#### What does this PR do?
Authored by @jcfilben. This should allow the Unsplash images to be displayed.

### CODE REVIEW

File: aries-site/src/components/seo/Meta.js [hunk 1]
```diff
@@ -93,11 +93,11 @@ export const Meta = ({ title, description, canonicalUrl, socialImageUrl }) => {
         httpEquiv="Content-Security-Policy"
         content="default-src 'self' 'unsafe-eval'; 
         style-src 'self' *.hpe.com/hfws-static/slim/css/ 'unsafe-inline';
-        connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com https://eyes.applitools.com;
+        connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com;
         media-src 'self' https://d3hq6blov2iije.cloudfront.net/media/HPE+Design+System-v3.mp4;
-        img-src 'self' https://www.google-analytics.com;
+        img-src 'self' https://www.google-analytics.com https://images.unsplash.com/ http://s.gravatar.com/avatar/;
```
Comment (halocline) [inline aries-site/src/components/seo/Meta.js:98]: 👍

File: aries-site/src/components/seo/Meta.js [hunk 2]
```diff
@@ -93,11 +93,11 @@ export const Meta = ({ title, description, canonicalUrl, socialImageUrl }) => {
...
         script-src-elem 'self' *.hpe.com https://www.google-analytics.com/analytics.js;
-        font-src *.hpe.com hpefonts.s3.amazonaws.com *.cloudfront.net/fonts/;
+        font-src *.hpe.com hpefonts.s3.amazonaws.com https://d3hq6blov2iije.cloudfront.net/fonts/;
```
Comment (halocline) [inline aries-site/src/components/seo/Meta.js:100]: 👍

File: aries-site/src/components/seo/Meta.js [hunk 3]
```diff
@@ -93,11 +93,11 @@ export const Meta = ({ title, description, canonicalUrl, socialImageUrl }) => {
         httpEquiv="Content-Security-Policy"
         content="default-src 'self' 'unsafe-eval'; 
         style-src 'self' *.hpe.com/hfws-static/slim/css/ 'unsafe-inline';
-        connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com https://eyes.applitools.com;
+        connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com;
```
Comment (halocline) [inline aries-site/src/components/seo/Meta.js:96]: 👍

**Review by halocline — APPROVED** [review]
Looks good and like the tightening up of the valid sources.

The build breaking is due to an unrelated issue.

**Comment by britt6612** [comment]
Looks good I think this takes care of both of these issues #1509 #1500

**Pull request merged**